string_assignment.js

```diff
@@ -0,0 +1,153 @@
+// # String Exercises
+//
+// 1. `console.log` the first character of a string.
+let leaguer = "boyd";
+console.log(leaguer[0]);
+// 2. `console.log` the length of a string.
+console.log(leaguer.length);
+// 3. `console.log` the last character of any string.
+console.log(leaguer[leaguer.length-1]);
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+// 'jimmy'
+// // => 'jimmY'
+console.log(leaguer[0] + leaguer[1] + leaguer[2] + leaguer[leaguer.length-1].toUpperCase());
+// 5. Create a drEvil code block that will take a single number variable, and log the '<variableAmount> dollars',
+// your code block should add '(pinky)' at the end of the amount if it's 1 million. For example:
+//
+// ```js
+// let amount = 10
+// // => 10 dollars
+// let amount = 1000000
+// // =>  1000000 dollars (pinky)
+// ```
+
+let variableAmount = 18;
+let drEvil = variableAmount + " dollars";
+  if (variableAmount === 1000000) {
+    console.log(drEvil + " (pinky)");
+  } else {
+    console.log(drEvil);
+  }
+
+// 6. Create a `verbing` code block. It should take a single string variable. If its length is at least 3,
+// it should add 'ing' to its end, unless it already ends in 'ing', in which case it should add 'ly' instead.
+// If the string length is less than 3, it should leave it unchanged.
+// For example:
+//
+// ```js
+// verbing('box')
+// // => 'boxing'
+// verbing('train')
+// // => 'training'
+// verbing('swimming')
+// // =>  'swimmingly'
+// verbing('go')
+// // =>  'go'
+// ```
+
+let myString = "grow";
+if (myString.length >= 3 && myString.endsWith("ing")) {
+  console.log(myString + "ly");
+} else if (myString.length >= 3) {
+  console.log(myString + "ing")
+} else {
+  console.log(myString);
+}
+
+
+// 7. Create a `withoutLast` code block that takes a single string variable, and returns a copy of the string without the last word.
+let weatherToday = "partly cloudy sunny";
```

Comment:
What would happen if we change weatherToday to "partly cloudy sunny stormy"? Current code only gives us "party cloudy" instead of "partly cloudy sunny". You almost got the answer here and need only small tweak.